mmv1/products/transcoder/JobTemplate.yaml

```diff
@@ -0,0 +1,1137 @@
+                          default_from_api: true
+      - !ruby/object:Api::Type::Array
+        name: "editList"
+        description: |
+          List of input assets stored in Cloud Storage.
+        default_from_api: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: "key"
+              description: |
+                A unique key for this atom.
+              default_from_api: true
+            - !ruby/object:Api::Type::Array
+              name: "inputs"
+              item_type: Api::Type::String
+              description: |
+                List of values identifying files that should be used in this atom.
+              default_from_api: true
+            - !ruby/object:Api::Type::Time
+              name: "startTimeOffset"
+              description: |
+                Start time in seconds for the atom, relative to the input file timeline.
+              default_from_api: true
+            - !ruby/object:Api::Type::Time
+              name: "endTimeOffset"
+              description: |
+                End time in seconds for the atom, relative to the input file timeline.
+              default_from_api: true
+      - !ruby/object:Api::Type::Array
+        name: "elementaryStreams"
+        description: |
+          List of input assets stored in Cloud Storage.
+        default_from_api: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: "key"
+              description: |
+                A unique key for this atom.
+              default_from_api: true
+            - !ruby/object:Api::Type::NestedObject
+              name: "videoStream"
+              description: |
+                Encoding of a video stream.
+              default_from_api: true
+              properties:
+                - !ruby/object:Api::Type::NestedObject
+                  name: "h264"
+                  description: |
+                    H264 codec settings
+                  default_from_api: true
+                  properties:
+                    - !ruby/object:Api::Type::Integer
+                      name: "widthPixels"
+                      description: |
+                        The width of the video in pixels.
+                      default_from_api: true
+                    - !ruby/object:Api::Type::Integer
+                      name: "heightPixels"
+                      description: |
+                        The height of the video in pixels.
+                      default_from_api: true
+                    - !ruby/object:Api::Type::Integer
+                      name: "frameRate"
+                      description: The target video frame rate in frames per second (FPS).
+                      required: true
+                    - !ruby/object:Api::Type::Enum
+                      name: "frameRateConversionStrategy"
```

Comment:
I'm noticing that a lot of fields have default values yet do not mention it in the description. Could you include them in? VideoStreams.h264 has a good amount of fields that could include more info in regards to the values. https://cloud.google.com/transcoder/docs/reference/rest/v1/JobConfig#H264CodecSettings

For example, the default value of `frameRateConversionStrategy` is `DOWNSAMPLE` from the docs:

> Optional. Frame rate conversion strategy for desired frame rate. The default is DOWNSAMPLE.